aperturedb/Utils.py

```diff
@@ -112,7 +112,7 @@ def get_schema(self, refresh=False):
             }
         }]
 
-        res, _ = self.client.query(query)
+        res, _ = self.execute(query)
```

Comment:
I think this will change the behaviour slightly, but not necessarily in a bad way.